### ISSUE TITLE
Probes don't connect the control WS to multiple apps

### ISSUE THREAD

**Issue by 2opremio** [weaveworks/scope#698]
I launched my probe against an app with:

```
scope launch --service-token=ejj1cm4z161nbqp5e6iqpfo5krpwixbm 10.0.0.60:80
```

Which should result in the probe connecting to both `10.0.0.60` (where my app lives) and `scope.weave.works:80` (it should probably not connect to `scope.weave.works:80` ... but that's a different story)

```
docker exec weavescope ps | grep scope-probe
29262 root       0:24 /home/weave/scope-probe -docker=true -token=ejj1cm4z161nbqp5e6iqpfo5krpwixbm scope.weave.works:80 10.0.0.60:80
```

When trying to use the new container playback controls (e.g. pause, stop ...) I got a 404 in the UI and noticed the following error in the logs:

```
<app> 2015/11/25 14:04:11 Probe 45a13e4aba497af0 is not connected right now…
```

To debug whether the probe was connecting the control websocket to the app I added this line after https://github.com/weaveworks/scope/blob/master/xfer/app_client.go#L90

```
log.Printf("CONTROL ENDPOINT %s", url)
```

Resulting in 

```
$ docker logs weavescope |& grep CONTROL
<probe> 2015/11/25 15:28:47 CONTROL ENDPOINT ws://52.91.255.127:80/api/control/ws
```

where `52.91.255.127` is the IP of `scope.weave.works`

```
$ host scope.weave.works | grep 52.91
docker.cloud.weave.works has address 52.91.255.127
```

However, there is no sign of `10.0.0.60`, which does appear in the logs if I execute the probe manually with just one app target:

```
$ prog/probe/scope-probe -docker=true -token=ejj1cm4z161nbqp5e6iqpfo5krpwixbm 10.0.0.60:80 |& grep CONTROL
<probe> 2015/11/25 15:36:53 CONTROL ENDPOINT ws://10.0.0.60:80/api/control/ws
```

**Comment by 2opremio** [comment]
As a workaround, you can remove `echo "scope.weave.works:80" >/etc/weave/apps` from `docker/entrypoint.sh` and rebuild the scope image.

**Comment by tomwilkie** [comment]
What ids do both apps show? I've what's in /api?

**Comment by 2opremio** [comment]
what ids? The probe ids? I can get you the `/api` endpoint later today.

BTW scope.weave.works runs 0.8.0 and 10.0.0.60 was running 0.10.0

**Comment by tomwilkie** [comment]
The app ids.

**Comment by 2opremio** [comment]
OK.

* For scope.weave.works: `{"id":"78a72a583a3c28ce","version":"0.8.0"}` although the probe cannot access it since it its token corresponds to the the other app.
* For `10.0.0.60`: ` {"id":"52de2de4e16dd1e5","version":"0.10.0"}

**Comment by tomwilkie** [comment]
Is 10.0.0.60 a service instance running locally?

**Comment by tomwilkie** [comment]
I get this when I visit scope.weave.works/api:

```
{"id": "sccopeservice", "version": "0.1"}
```

If you;re running another instance of the service locally, then it will
have the same ID as the real service, and the probe will only send reports
/ connect controls ws to one of them.  Unfortunately it can current pick
one for the reports and another one for controls, which needs fixing, but
it already being tracked in a separate issue.

**Comment by 2opremio** [comment]
> Is 10.0.0.60 a service instance running locally?

Yes, what I pasted before is the `api/app/{orgName}/api` endpoint of the app within the service

> I get this when I visit scope.weave.works/api:
> ```
{"id": "sccopeservice", "version": "0.1"}
```

The `/api` endpoint of the service (which now I realize is all the probes really care about) is also  `{"id": "sccopeservice", "version": "0.1"}` in both cases

> If you;re running another instance of the service locally, then it will
have the same ID as the real service, and the probe will only send reports
/ connect controls ws to one of them.  Unfortunately it can current pick
one for the reports and another one for controls, which needs fixing, but
it already being tracked in a separate issue.

What issue? I cannot find it. Let's close this one as a duplicate then

**Comment by tomwilkie** [comment]
Sorry for the delay: once we fix #630 this should go away.
